stock_location_analytic_mrp/views/mrp.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Deactive duplicate view for field analytic_account_id -->
```

Comment:
Why it is duplicated? if the duplicated view is in our modules perhaps we can update that view instead.

Reply:
one is OCA module and one is from base module so we can't change anything there I think

Reply:
ok! LGTM can you just rebase to see if CI is green before merging?

Reply:
it depends on https://github.com/ForgeFlow/eficent-odoo-addons/pull/294, so need to merge first it

Reply:
yes, my bad, sorry

Reply:
Just merge the dependency :)